faucet/app.py

```diff
@@ -14,11 +14,16 @@
 from slowapi import Limiter, _rate_limit_exceeded_handler
 from slowapi.errors import RateLimitExceeded
 from slowapi.util import get_remote_address
+from typing import list
+from tokenlists import TokenListManager
+from ape import accounts
 
 limiter = Limiter(key_func=get_remote_address)
 app = FastAPI()
 app.state.limiter = limiter
 app.add_exception_handler(RateLimitExceeded, _rate_limit_exceeded_handler)
+tlm = TokenListManager()
+tlm.install_tokenlist("tokens.1inch.eth")
```

Comment:
Is this the list we want to use?

Reply:
This list seems to work

Reply:
I think it works better with the uniswap list which provides multiple chains at a time.